azure-pipelines.yml

```diff
@@ -39,11 +44,15 @@ jobs:
           $(EXTRA_CMAKE_ARGS)
           $(Build.SourcesDirectory)
 
-    - task: CMake@1
-      displayName: "Check code formatting"
-      inputs:
-        workingDirectory: $(Build.BinariesDirectory)/build
-        cmakeArgs: --build . --target format_check
+    # Temporary; this is disabled because it complains about the config.h files
```

Comment:
We need to enable this back on, but we should prevent generated config files from causing this check to fail

Reply:
I think that the easiest way is to add an ignore folder option (if it's not already there) in the script that checks the format.
We can ignore the entire libraries folder and be done with it.
Though if we want to have it more flexible, we should also probably add an option to specify single files.